lighthouse-core/test/lib/asset-saver-test.js

```diff
@@ -96,6 +96,7 @@ describe('asset-saver helper', () => {
           {args: {IsMainFrame: true}, cat: 'v8', pid: 1, ts: 5},
           {args: {data: {encodedDataLength: 20, requestId: '1.22'}}, pid: 1, ts: 6},
         ],
+        metadata: {'cpu-model': 9001, 'network-type': 'Unknown'},
```

Comment:
not directly important to Lighthouse, but overlooked in previous testing of `saveTrace()` and very relevant to where the various methods are putting newlines within their output :)